webrender/src/frame_builder.rs

```diff
@@ -1345,6 +1345,7 @@ impl FrameBuilder {
             glyph_gpu_blocks: Vec::new(),
             glyph_keys: Vec::new(),
             offset: run_offset,
+            shadow_color: ColorU::new(0, 0, 0, 0),
```

Comment:
Would be best (but not required) to have the `TextRunPrimitiveCpu` in a separate commit, since it's technically independent of the mask swizzling.